ts/components/RemindEmailValidationOverlay.tsx

```diff
@@ -303,39 +307,53 @@ class RemindEmailValidationOverlay extends React.PureComponent<Props, State> {
 
   public render() {
     const email = this.props.optionEmail.getOrElse(EMPTY_EMAIL);
+
     const { isOnboardingCompleted } = this.props;
-    const image = this.state.emailHasBeenValidate
-      ? require("../../img/email-checked-icon_ok.png")
-      : require("../../img/email-checked-icon.png");
+
+    const icon = this.state.emailHasBeenValidate
+      ? "io-email-validated"
+      : "io-email-to-validate";
+
     const title = this.state.emailHasBeenValidate
       ? I18n.t("email.validate.validated")
       : I18n.t("email.validate.title");
+
     return (
       <TopScreenComponent
         {...(isOnboardingCompleted ? this.onMainProps : this.onBoardingProps)}
         contextualHelp={this.contextualHelp}
       >
-        <Content>
-          <React.Fragment>
-            <Image style={styles.imageChecked} source={image} />
-            <View spacer={true} extralarge={true} />
-          </React.Fragment>
-          <H2 style={isOnboardingCompleted ? styles.emailTitle : undefined}>
+        <Content bounces={false}>
+          <View spacer={true} extralarge={true} />
+          <IconFont
+            name={icon}
+            size={84}
+            color={customVariables.brandHighlight}
+            style={styles.center}
+          />
+          <View spacer={true} extralarge={true} />
+          <Text alignCenter={true} bold={true}>
             {title}
-          </H2>
-          <View spacer={true} />
+          </Text>
           {!this.state.emailHasBeenValidate ? (
-            <Markdown onLoadEnd={this.handleOnContentLoadEnd}>
+            <Markdown
+              onLoadEnd={this.handleOnContentLoadEnd}
+              cssStyle={"body { text-align: center;}"}
```

Comment:
could it be a const?

Reply:
updated! https://github.com/pagopa/io-app/pull/1658/commits/d8892ad036d58806a7f930d89aacb8b451655d2b